forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -66,20 +66,20 @@ internal class ConfigHelper {
 
             return FROptionsBuilder.build {
                 server {
-                    url = sharedPreferences.getString(ConfigHelper.url, null) ?: ""
-                    realm = sharedPreferences.getString(ConfigHelper.realm, null) ?: ""
-                    cookieName = sharedPreferences.getString(ConfigHelper.cookieName, null) ?: ""
+                    url = sharedPreferences.getString(ConfigHelper.url, null) ?: context.getString(R.string.forgerock_url)
```

Comment:
Can we use the defValue sharedPreferences.getString(value, defValue) for the null case.
sharePreferences.getString(ConfigHelper.url, context.getString(R.string.forgerock_url)

Reply:
no change required. already discussed